extensions/VisualEditor/wikia/modules/ve/tests/ce/ve.ce.wikiaExample.js

```diff
@@ -104,7 +104,17 @@ ve.ce.wikiaExample = ( function ( utils ) {
 
 	media.html.video = {
 		playButton:
-			'<span class="play-circle ve-no-shield"></span>'
+		'<span class="thumbnail-play-icon-container">' +
```

Comment:
There should be `ve-no-shield` I think...

Reply:
this class was not used in css at all